stable/neo4j/templates/core-statefulset.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         env:
           - name: NEO4J_dbms_mode
             value: CORE
+          - name: NEO4J_ACCEPT_LICENSE_AGREEMENT
+            value: "yes"
```

Comment:
I think having this default value in here is problematic. I'm not aware of the actual license involved, but it's generally not a good idea to accept the license for all our end users. It's better if this is moved to a variable that defaults to "no", so an end user has to specifically accept the license himself by overriding that parameter. I think that's a safer path.

That variable should then also be added to the README, of course.

Reply:
@timstoop I understand the concern you're raising; on the other side if it defaults to no, a user must clone the repo and modify locally, and the chart isn't directly runnable, which is not desirable.

Would it be acceptable if I added a notice front and center in the README that launching from the repo constitutes acceptance of the license, link this portion of the code for anyone who wishes to change it?

If that isn't acceptable, I understand but I may close the PR, as this change is a key one that makes the containers "just work" out of the box.

Reply:
No, a user would not have to clone the repo, the whole idea of variables is that they can be easily overridden by a user :) If you give the variable a clear name like `iAcceptTheLicense` (someone else might have a better suggestion here), they can do something like `helm install stable/neo4j --set iAcceptTheLicence=yes` and the variable would be set correctly. I don't know about you, but we generally keep custom config.yaml for each deployment anyway, so we can version those in Git and easily override the defaults. It's a simple change that does not require the kubernetes project to accept the license on behalf of all users.

Reply:
I'm highly opposed to default it to yes. 
The agreement has to be explicitly accepted by the users.

Reply:
The neo4j community edition is based on GPLv3 and I've never seen a user have to explicitly accept that. However, this chart is based on the enterprise edition, which carries some restrictions (see [licensing](https://neo4j.com/licensing/)).

Reply:
This is the same issue as the Minecraft chart, I believe, and there the default is "no" and if you don't override the default "false" on the `eula` value the chart does not actually deploy the app and complains at you (via NOTES.txt): https://github.com/kubernetes/charts/blob/master/stable/minecraft/templates/NOTES.txt